install_pim/manual/crontab_tasks.rst.inc

```diff
@@ -16,15 +16,17 @@ You have to add these 2 commands to your local installation:
 
     php /path/to/installation/pim-community-standard/bin/console pim:completeness:calculate --env=prod    # recalculates the products completeness
     php /path/to/installation/pim-community-standard/bin/console pim:versioning:refresh --env=prod        # processes pending versions
+    php /path/to/installation/pim-community-standard/bin/console pim:volume:aggregate --env=prod          # recalculates the aggregate volumes
```

Comment:
Just at the bottom L12 (https://github.com/akeneo/pim-docs/pull/925/files#diff-f1ea729a896110145fbf6bd8be8feba7R12), we talk about 2 commands so you need to update

Reply:
I also think you should explain what is "recalculates the aggregate volumes". I work at Akeneo and I don't know what is it so from an external point of view, you can be totally lost

Reply:
I change the number & change the description to "recalculates the aggregate volumes shown on volume screen" => @fitn do you better understand ?